addons/gut/gut_cmdln.gd

```diff
@@ -346,10 +346,10 @@ func _run_gut():
 			load_options_from_config_file(opt_resolver.get_value('config_file'), opt_resolver.config_opts)
 
 	if(load_result == -1): # -1 indicates json parse error
-		quit()
+		quit(-1)
 	else:
 		if(!all_options_valid):
-			quit()
+			quit(-1)
```

Comment:
`-1` isn't a valid exit code.  When you pass any value less than 0 to `quit` it will use `OS.exit_code`.  So far GUT only has exit codes for success(0)/fail(1) so change these to be `quit(1)`.